tests/stdlib/tstrutil.nim

```diff
@@ -7,6 +7,14 @@ discard """
 import
   strutils
 
+import macros
```

Comment:
Why is `macros` imported?

Reply:
``astToStr`` I guess

Reply:
Yup. Or is there a simpler way to echo the code handed to the template?

Reply:
``astToStr`` is in system!

Reply:
Ooops, sorry! Please remove that import... :/